### PR TITLE
Release 2.1.4

### PR DESCRIPTION
**Pull request by Tassosb**
PR #238 unblocks me. I'd like to get it out in a release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
 
 * ...
 
+## 2.1.4 2022-07-05
+
+* Truncate: Simplify VALUES(...) lists
+* Truncate: Correctly handle UPDATE and ON CONFLICT target lists
+* Support complex queries with deeply nested ASTs ([#238](https://github.com/pganalyze/pg_query/pull/238))
```
Comment (lfittl) [inline CHANGELOG.md:11]: @Tassosb We should also add the following two that were added after this PR was opened:

```
* Find table references inside type casts
* Find function calls referenced in expression indexes
```

File: Gemfile.lock [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+    pg_query (2.1.4)
       google-protobuf (>= 3.19.2)
 
 GEM
   remote: https://rubygems.org/
   specs:
     ast (2.4.1)
     diff-lcs (1.3)
-    google-protobuf (3.20.0)
+    google-protobuf (3.21.2)
```
Comment (lfittl) [inline Gemfile.lock:12]: I did this in a separate PR (and to a slightly newer version) for clarity: https://github.com/pganalyze/pg_query/pull/253/files

**Comment by lfittl** [comment]
> PR #238 unblocks me. I'd like to get it out in a release.

Thanks for your patience here - after we get the changelog updated (see my comment), and #253 merged, we should be able to get this release out.

**Comment by nicmarti** [comment]
Any update on this release ?

**Comment by stanhu** [comment]
https://github.com/pganalyze/pg_query/pull/257

**Comment by lfittl** [comment]
Done in #257 and released to Rubygems. Thanks @stanhu for the help!

**Pull request closed without merging**